sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ClientEncryptionIncludedPath.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Path that needs encryption and the associated settings within {@link ClientEncryptionPolicy}.
+ */
+public final class ClientEncryptionIncludedPath {
```

Comment:
any class in `com.azure.cosmos.models` is in public API.

Aside from the fact that if this needs to be in the azure-cosmos or in the encryption project

you have public fields in this class, I think you need getter/setter in the public api also perhaps we should annotate all the classes and methods related to encryption in the azure-cosmos with beta api?

Reply:
As discussed, this PR we will not include the work of introducing api with ObjectNode in cosmos. I added getter setter with beta annotation